tests/virtualization/virt_install.pm

```diff
@@ -19,7 +19,8 @@ sub run {
     x11_start_program('xterm');
     become_root;
     script_run('virt-install --name TESTING --memory 512 --disk none --boot cdrom --graphics vnc &', 0);
-    x11_start_program('vncviewer :0', target_match => 'virtman-gnome_virt-install', match_timeout => 200);
+    wait_still_screen;
```

Comment:
Keep in mind that the default for `wait_still_screen` is to have a still time of 7 seconds so that means we waste at least 7 seconds in every test which I consider too much in general. Can we please try to lower this time? Something <= 3 is ok or we need to find a different approach using something else